lib/wsapp.js

```diff
@@ -12,15 +12,81 @@ var MessageBroker = require('./messagebroker');
 
 log.level = 'debug';
 
-var WsApp = function() {};
-
-WsApp.prototype._unauthorized = function(socket) {
-  socket.emit('unauthorized');
-  socket.disconnect();
+var WsApp = function() {
+    this.connectedWallets = {};
+    this.authorizedWallets = {};
 };
 
+function keysToArray(dict){
```

Comment:
You may want to take a look at [`_.keys()`](https://lodash.com/docs#keys) from lodash.